pkg/cli/connect.go

```diff
@@ -74,7 +73,7 @@ func connectCmdRun(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func connectToFileNode(node *service.FileNode, args []string) error {
+func connectToFileNode(that *service.FileNode, args []string) error {
 	for _, nID := range args {
```

Comment:
- What is args? 
- Parsing nodes from the args is duplicated in the connectToPackageNode function.
- When parsing fails there might be nodes added (no transaction)

Proposed solution:
Parse all args then descend into the the connect functions, that have get passed a proper slice of nodes (or interface{})